backend/api/src/config.rs

```diff
@@ -16,6 +16,9 @@ pub struct AppConfig {
     /// The database URL to use
     pub database_url: String,
 
+    /// The redis URL to use
+    pub redis_url: String,
```

Comment:
We also make use of Redis Sentinel.

Redis Sentinel is a way to make a HA Redis Cluster. Since the master switches to a slave if it becomes unresponsive or is shutting down.

So perhaps have an option to enable sentinel and instead convert this to 
```
redis_urls: Vec<String>,
redis_username: String,
redis_password: String,
redis_sentinel: bool,
```